src/matrixHelpers/FormioScript/index.js

```diff
@@ -64,11 +64,11 @@ export const getDefaultScripts = ({ subdomain, version = defaultVersion }) => {
       href: `https://${subdomain}.qgov.net.au/formio-qld/${version}/premium.css`,
       rel: "stylesheet",
     },
-    {
-      type: "link",
-      href: `https://${subdomain}.qgov.net.au/formio-qld/${version}/formio-qld.min.css`,
-      rel: "stylesheet",
-    },
+    // {
```

Comment:
why ignore this at this time?

Reply:
the style has been moved to SWE library with another ticket by asif
in order to have the same style for both swe form and formio form